travis-ci/vmtest/run.sh

```diff
@@ -426,11 +426,12 @@ if [[ ! -z SETUPCMD ]]; then
 	setup_script=$(printf "#!/bin/sh
 set -eux
 dmesg -c
+> /exitstatus
```

Comment:
is this to create a file? quite unusual, I usually do `touch <file>` for that

Reply:
No, this is to truncate it, in the (admittedly unlikely) case it's already existing. I don't see it so often, although I thought it was rather admitted as a classic way to truncate a file. I can so some research or change to `printf '' > /exitfile` if you prefer. Or add a comment.

Reply:
Oh, truncation. Yeah, it's fine, but let's add a brief comment.